src/MarketingPageAcceptanceTests.TestData/Utils/Queries.cs

```diff
@@ -2,30 +2,30 @@
 
-        internal const string GetMarketingContacts = "SELECT FirstName, LastName, PhoneNumber, Department, Email FROM MarketingContact WHERE SolutionId=@solutionId";
-        internal const string GetNumberOfMarketingContactsForSolution = "SELECT COUNT (*) AS Count FROM [dbo].[MarketingContact] WHERE SolutionId=@solutionId";
+        internal const string GetMarketingContacts = "SELECT * FROM MarketingContact WHERE SolutionId=@solutionId";
+        internal const string GetNumberOfMarketingContactsForSolution = "SELECT COUNT(*) FROM [dbo].[MarketingContact] WHERE SolutionId=@solutionId";
         internal const string CreateMarketingContact = "INSERT INTO [MarketingContact] (SolutionId, FirstName, LastName, Email, PhoneNumber, Department, LastUpdated, LastUpdatedBy) VALUES(@solutionId, @firstName, @lastName, @email, @phoneNumber, @department, GETDATE(), '00000000-0000-0000-0000-000000000000')";
         internal const string DeleteMarketingContact = "DELETE FROM MarketingContact where SolutionId=@solutionId";
 
         internal const string UpdateLastUpdated = "UPDATE @table SET LastUpdated=@lastUpdated WHERE @whereKey=@whereValue";
         internal const string GetLastUpdated = "SELECT LastUpdated FROM @table WHERE @whereKey=@whereValue";
 
-        internal const string CreateNewSupplier = "INSERT INTO Supplier (Id, Name, LegalName, Summary, SupplierUrl, LastUpdated, LastUpdatedBy) values (@supplierId, @name, @legalName, @summary, @supplierUrl, @lastUpdated, @lastUpdatedBy)";
-        internal const string DeleteSupplier = "DELETE from Supplier where Id=@supplierId";
-        internal const string GetSupplier = "SELECT [Id],[Name],[Summary],[SupplierUrl],[LastUpdated],[LastUpdatedBy]FROM [dbo].[Supplier] WHERE [Id]=@supplierId";
+        internal const string CreateNewSupplier = "INSERT INTO Supplier (Id, Name, LegalName, Summary, SupplierUrl, LastUpdated, LastUpdatedBy) values (@Id, @name, @legalName, @summary, @supplierUrl, @lastUpdated, @lastUpdatedBy)";
+        internal const string DeleteSupplier = "DELETE from Supplier where Id=@Id";
+        internal const string GetSupplier = "SELECT * FROM [dbo].[Supplier] WHERE [Id]=@Id";
         internal const string GetSupplierForSolution = "SELECT Supplier.[Id],Supplier.[Name],Supplier.[Summary],Supplier.[SupplierUrl],Supplier.[LastUpdated],Supplier.[LastUpdatedBy]FROM [dbo].[Supplier] LEFT JOIN Solution on Solution.SupplierId = Supplier.Id WHERE Solution.Id=@solutionId";
-        internal const string UpdateSupplier = "UPDATE Supplier SET Name=@name, LegalName=@legalName, Summary=@summary, SupplierUrl=@supplierUrl, LastUpdated=@lastUpdated, LastUpdatedBy=@lastUpdatedBy WHERE Id=@supplierId";
+        internal const string UpdateSupplier = "UPDATE Supplier SET Name=@name, LegalName=@legalName, Summary=@summary, SupplierUrl=@supplierUrl, LastUpdated=@lastUpdated, LastUpdatedBy=@lastUpdatedBy WHERE Id=@Id";
     }
```

Comment:
Since these queries are only used by their specific object class, should we move them into their and delete this file?

Reply:
Definitely something to consider for future refactoring.